rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java

```diff
@@ -136,6 +136,8 @@
 
     private ScriptConsoleView scriptConsoleView = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "scriptConsoleView" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)